code/socialDistribution/views.py

```diff
@@ -42,12 +44,12 @@ def loginPage(request):
 
             if user is not None:
                 login(request, user)
-                return redirect('socialDistribution:home', author_id=author_id)
+                return redirect(f'/author/{author_id}/home/', author_id=author_id)
```

Comment:
I think these do the same thing. Right?
If so let's use the pathname socialDistribution:home  instead of the actual path